webpack/routes/AnsibleRoles/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import { AnsibleRolesPage } from '../../components/AnsibleRoles/AnsibleRolesPage';
+
+const AnsibleRoles = () => (
+  <React.Fragment>
+    <Helmet>
+      <title>{__('Ansible Roles')}</title>
+    </Helmet>
+    <AnsibleRolesPage />
+  </React.Fragment>
+);
```

Comment:
No need to wrap it and add Helmet as its provided by TableIndexPage

Reply:
Removed